src/RestController.php

```diff
@@ -998,4 +973,34 @@ protected function createHalEntity($entity)
             $this->getRouteIdentifierName()
         );
     }
+
+    /**
+     * @param Exception|Throwable $error
+     * @return ApiProblem
+     */
+    private function createApiProblem($error)
+    {
+        return new ApiProblem(
+            $this->validateHttpStatusCode($error->getCode()),
+            $error->getMessage()
+        );
+    }
+
+    /**
+     * Ensure we have a valid HTTP status code for an ApiProblem
+     *
+     * @param int|string $code
+     * @return int
+     */
+    private function validateHttpStatusCode($code)
```

Comment:
Just like the `createApiProblem()` notes, just re-instate the original method, but drop the typehint from the method signature and update the docblock to indicate what types are allowed; this is necessary to preserve backwards compatibility, and to allow users to re-use these methods in their extending classes.